packages/paste-core/components/checkbox/src/Checkbox.tsx

```diff
@@ -25,6 +25,7 @@ export interface CheckboxProps extends React.InputHTMLAttributes<HTMLInputElemen
   isSelectAll?: boolean;
   isSelectAllChild?: boolean;
   checked?: boolean;
+  defaultChecked?: boolean;
```

Comment:
Adding the defaultChecked prop to help differentiate between a controlled component and an uncontrolled component. This follows what react recommends for uncontrolled inputs: https://reactjs.org/docs/uncontrolled-components.html#default-values